### PR TITLE
should be everything

### PR DESCRIPTION
**Pull request by crodeheaver**
Though I've been wrong before.

### CODE REVIEW

File: temperature.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+let Temperature = {
+    toFahrenheit: function (celsius) {
```
Comment (NathanBland) [inline temperature.js]: For using the rest of `es6` i'm confused as to why you used the `function` keyword here.

Comment (crodeheaver) [inline temperature.js]: Habit. I still default to es5 for some things. Good point though.

Comment (crodeheaver) [inline temperature.js]: `#fixed` `#needToFixBadEs5Habits` `#needToShortenMyHastags`

**Comment by NathanBland** [comment]
I am going to add that this doesn't meet all the requirements... Just an FYI :wink: 

Edit: I lied.

**Comment by crodeheaver** [comment]
I thought so. I was looking over it trying to figure out what I missed

**Comment by jgimbel** [comment]
Are you afraid of setting constants? Why use `let`?

**Comment by barberboy** [comment]
Ha!

**Comment by barberboy** [comment]
Sorry for closing the PR. I changed the default branch to be `gh-pages` and deleted `master`.

**Comment by crodeheaver** [comment]
@jgimbel Scared of const(s)? I'm down right terrified!

**Comment by barberboy** [comment]
The latest version is constiderably better than the first. I appreciate your constistency.

:100:

**Pull request closed without merging**